internal/services/kyc_service.go

```diff
@@ -0,0 +1,47 @@
+package services
+
```

Comment:
Please merge all of the kyc related files into this file, services package is an overkill (given it's only one service?)

Reply:
I would have preferred to have token-related methods and verifications-related methods in separate files, but I have applied your suggestion here.